Bugsnag/Breadcrumbs/BugsnagBreadcrumbs.m

```diff
@@ -197,3 +217,19 @@ - (void)deleteBreadcrumbFiles {
 }
 
 @end
+
+#pragma mark -
+
+void BugsnagBreadcrumbsWriteCrashReport(BSG_KSCrashReportWriter *writer) {
+    char path[PATH_MAX];
+    writer->beginArray(writer, "breadcrumbs");
+    for (unsigned int i = g_context.firstFileNumber; i < g_context.nextFileNumber; i++) {
+        int result = snprintf(path, sizeof(path), "%s/%d.json", g_context.directoryPath, i);
```

Comment:
```suggestion
        int result = snprintf(path, sizeof(path), "%s/%lu.json", g_context.directoryPath, i);
```
?

Reply:
Fixed in b2a806d792234e64dd73dd8d1b364eb6bee10095